packages/@react-spectrum/s2/stories/Tooltip.stories.tsx

```diff
@@ -78,6 +80,22 @@ export const Example = (args: any) => {
   );
 };
 
+Example.play = async ({canvasElement}) => {
+  await userEvent.tab();
+  let body = canvasElement.ownerDocument.body;
+  await within(body).findByRole('tooltip');
+};
```

Comment:
I removed the default `isOpen` on the tooltip stories because visible jank happens for the positioning of tooltips rendered out of view and if the container story div rendered a scroll bar and removed it during calculation of the overlay position. Ideally, the autodocs would not have `isOpen: true` so that I could still preserve the chromatic tests that snap shot the tooltips as open but I believe this isn't possible short of creating a completely custom docs page. 

Instead used storybook interactions test as a substitute since they don't affect the docs page